### DISCUSSION TITLE
the "base" attribute of the site configuration does not take effect

### DISCUSSION THREAD

**Discussion by qyingkou** [vuepress/core#1212] [Q&A] [answered]
## my confusion
After build the Vuepress site, I deploy the dist to repository's gh-page branch. However the problem is, the resource url can't add the value of the “base” attribute from site configuration（As shown in Figure 3）, but the document can be accessed（base value added, As shown in Figure 3).

I did study the configuration documentation carefully, and I did. (As shown in Figure 1)
Picture0: the description of the base configuration item on the official website.
<img width="797" alt="image" src="https://user-images.githubusercontent.com/46431594/209624384-16035a41-610b-4a6c-aae0-fd290f90070a.png">

## background
I'm installed  the latest version of Vuepress, "2.0.0-beta.59"
you can visit the problem site:
- [qyingkou.github.io/cungee/](https://qyingkou.github.io/cungee/)
- [qyingkou.github.io/cungee/markdown/](https://qyingkou.github.io/cungee/markdown/)

the repository here: 
- [cungee](https://github.com/qyingkou/cungee)

## configuration and code
`Picture1`   I have indicated the name of the repository.
<img width="737" alt="image" src="https://user-images.githubusercontent.com/46431594/209621261-a1c63dfc-6a6c-4f73-8680-3d20653c24fe.png">

Picture2：Use github action to build successfully, using the deployment code provided by vuepress official website.
<img width="977" alt="image" src="https://user-images.githubusercontent.com/46431594/209621851-eb5953db-d65e-4dea-b876-493750b052af.png">

Picture3：the resource request in the browser, obviously the request address is wrong, why?
<img width="1339" alt="image" src="https://user-images.githubusercontent.com/46431594/209621609-3fffabb7-7398-45b8-983b-938d8d36dde5.png">

Picture4: the document request in the browser, base attribute is valid here！why?
<img width="1131" alt="image" src="https://user-images.githubusercontent.com/46431594/209623604-0034d7a4-498e-4fba-8610-68b878033895.png">

Waiting for help, thank you！

**Comment by Mister-Hope** [comment, marked as answer]
> I did study the configuration documentation carefully

# ???

# Then why DOES you set config HERE?

![image](https://user-images.githubusercontent.com/33315834/209631512-6e85998f-6e15-4c86-83c1-21775ded3888.png)

- https://v2.vuepress.vuejs.org/reference/config.html
- https://v2.vuepress.vuejs.org/reference/default-theme/config.html

Read the above docs AGAIN, we do not pay for users' careless.
